clients/src/main/java/org/apache/kafka/clients/consumer/internals/Coordinator.java

```diff
@@ -317,7 +317,7 @@ public void handle(JoinGroupResponse joinResponse, RequestFuture<Void> future) {
                 log.debug("Joined group: {}", joinResponse.toStruct());
 
                 // record re-assignment time
-                sensors.partitionReassignments.record(response.requestLatencyMs());
```

Comment:
The issue is the lack of consistency. I don't mind which convention we pick, but we should be consistent in method names. Between `ClientRequest` and `ClientResponse`, we have (in trunk) `creationTime`, `receivedTime`, `getSendMs` and `requestLatencyMs`. This is clearly suboptimal. I picked one approach in the PR to start the discussion.